content/guides/getting-started/component-framework-configuration.md

```diff
@@ -47,12 +47,13 @@ cypress open
 The Launchpad's setup wizard will do the following things:
 
 1. Determine what changes need to be merged into your Cypress Config file.
+<!-- see GitHub comment-->
 2. Create a component support file for [configuring global styles]() and
```

Comment:
Empty link: `[configuring global styles]()`

Reply:
Took note and will fix when I figure out where it should link to